CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
+- `max_version`: this can be used to specify the maximum version of `libddwaf` required to support this evaluation primitive.
+
+Both fields follow the semantic versioning schema `x.y.z` without a `v` in front nor any subsequent labels or hashes, the minimum allowed version is `0.0.0` and the maximum `999.999.999`. Each new field can be provided in isolation or in combination with its counterpart. 
+
+The evaluation primitives supporting this new fields are: rules, exclusion filters, processors and scanners. An example of a rule using a minimum and maximum version can be seen below:
+
+```yaml
+  - id: rsp-930-004
+    name: SHi Exploit detection
+    tags:
+      type: shi
+      category: exploit_detection
+      module: rasp
+    min_version: 1.19.0
+    max_version  1.19.999
+    conditions:
+      - parameters:
+          resource:
+            - address: server.sys.shell.cmd
+          params:
+            - address: server.request.query
+        operator: shi_detector
+```
+
+Finally, when an evaluation primitive doesn't meet the required version criteria, its ID is included in a new diagnostic field called `skipped`, within the relevant section, e.g.
+```yaml
+
+rules:
+  skipped:
+    - rsp-930-004
+  loaded: ...
+```
+#### RASP operator versioning
+
+Finally, in order to distinguish multiple versions of our exploit prevention heuristics, RASP operators can now be versioned. Versioning is done with the following schema: `operator_name@version`, where the operator name is one of the existing RASP operators (`lfi_detector`, `ssrf_detector`, `sqli_detector`, `shi_detector`) and `version` consists of a single digit preceded by a `v`, e.g. `sqli_detector@v2`. 
+
+Operator versioning works as follows:
+- When the existing operator version is higher or equal to the required version, the available operator is compatible.
+- When the existing operator version is lower than the required version, the operator is incompatible.
+- When the operator is incompatible, the rule is silently skipped and added to the `skipped` section of the diagnostics.
+
+In addition, this release includes a new version of the `sqli_detector` operator, specifically `sqli_detector@v2`.
+
+### Release changelog
+#### Changes
+- Fingerprint regeneration based on availability of optional arguments ([#331](https://github.com/DataDog/libddwaf/pull/331))
+- Expand detections per parameter ([#332](https://github.com/DataDog/libddwaf/pull/332))
+- Extend exists operator to support key paths and negation ([#334](https://github.com/DataDog/libddwaf/pull/334))
+- Negated scalar condition for matchers ([#335](https://github.com/DataDog/libddwaf/pull/335))
+- Greater and lower than matchers  ([#336](https://github.com/DataDog/libddwaf/pull/336))
+- Support min_version and max_version on evaluation primitives and RASP operator versioning ([#343](https://github.com/DataDog/libddwaf/pull/343))
+- Introduce `sqli_detector@v2` ([#343](https://github.com/DataDog/libddwaf/pull/343))
```

Comment:
Is this link ok?

Reply:
Yes, I just wanted this change to be explicit